src/cpp/session/resources/grid/gridstyles.css

```diff
@@ -317,3 +317,15 @@ div.DTS div.DTS_Loading.showLoading {
     display: block;
 }
 
+.columnTypeHeader {
+    font-weight: normal;
+    font-style: italic;
+    color: #abacad;
+    float: right;
+}
+
+.columnClickable {
+    cursor: pointer;
+    background: url("datatables/images/header_expand.png") no-repeat right top;
+    padding-right: 15px !important;
```

Comment:
We generally try to make a rule more specific to create an overriding style (since `!important` is itself difficult to override), but this is okay if there's no other straightforward way to apply the style.